daemon/cmd/daemon_main.go

```diff
@@ -335,7 +335,10 @@ func init() {
 	flags.String(option.EgressMasqueradeInterfaces, "", "Limit egress masquerading to interface selector")
 	option.BindEnv(option.EgressMasqueradeInterfaces)
 
-	flags.Bool(option.EnableHostReachableServices, false, "Enable reachability of services for host applications")
+	flags.Bool(option.SkipSkLBInPodNS, false, "Skip socket lb for services when inside a pod namespace, in favor of service lb at the pod interface. Socket lb still used when in the host namespace.")
```

Comment:
I would prefer if the flag name wasn't shortened or abbreviated. Might as well leave it longer for more context rather than risk the terseness causing confusion. 

```suggestion
	flags.Bool(option.SkipSkLBInPodNS, false, "Skip socket LB for Services when inside a pod namespace, in favor of Service LB at the pod interface. Socket LB still used when in the host namespace.")
```

Reply:
Done.